lib/migrations.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2016 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import importlib
+from glob import glob
+
+from . import console
+from . import constants as const
+from .exceptions import BossManageError
+
+class MigrationFile(object):
+    def __init__(self, filepath):
+        self.filepath = filepath
+
+        module = filepath.replace(const.repo_path(), '')[1:]
+        module = os.path.splitext(module)[0]
+        module = module.replace('/', '.')
+        self.module_path = module
+
+        filename = module.rsplit('.', 1)[1]
+        vers, self.name = filename.split('_', 1)
+        self.start, self.stop = int(vers[0:4]), int(vers[4:8])
```

Comment:
Is the naming convention for migration files documented anywhere besides what's done in the code here?

Reply:
Thanks for catching that. I meant to document that

Reply:
Added a README in the migrations directory